homeassistant/components/traccar_server/coordinator.py

```diff
@@ -187,7 +187,7 @@ async def handle_subscription_data(self, data: SubscriptionData) -> None:
             self.data[device_id]["attributes"] = attr
             self.data[device_id]["geofence"] = get_first_geofence(
                 self._geofences,
-                position["geofenceIds"] or [],
+                get_geofence_ids(device, position),  # pylint: disable=undefined-loop-variable
```

Comment:
If you need to disable the linger, the code is not correct and should be adjusted instead of being disabled.

Reply:
Good catch, that was a wrong copy/paste of the function call